docker-compose.yml

```diff
@@ -1,13 +1,19 @@
 version: "3.3"
+networks:
```

Comment:
This is not needed for a general use case. Something in your setup is requiring this